packages/cursorless-engine/src/test/fixtures/talonApi.fixture.ts

```diff
@@ -131,8 +130,26 @@ export const talonApiFixture = [
     wrapWithSnippetByNameAction,
   ),
   spokenFormTest("parse tree air", parseTreeAction),
-  spokenFormTest("test api get text air", getTextAction),
-  spokenFormTest("test api get text list air", getTextListAction),
+  multiActionSpokenFormTest(
```

Comment:
How is this a multi action spoken form? All it does is getting the text in a single action?

Reply:
yeah that function is poorly named. That function has an arg that lets us mock the output of `get_text`, which is necessary for the test to work, as otherwise the `[0]` in python action breaks. Shall I rename the function / make a different function? I agree it's a bit confusing

Reply:
Yeah please rename it because I was thoroughly confused :D

Reply:
better?

Reply:
Yes